### ISSUE TITLE
Cannot build with cargo build

### ISSUE THREAD

**Issue by calixteman** [tree-sitter/tree-sitter#353]
I get this error:
```bash
   Compiling tree-sitter-cli v0.15.2 (/home/calixte/dev/mozilla/tree-sitter/cli)
error: couldn't read cli/src/../../lib/binding_web/tree-sitter.js: No such file or directory (os error 2)
  --> cli/src/web_ui.rs:14:31
   |
14 | const LIB_JS: &'static [u8] = include_bytes!("../../lib/binding_web/tree-sitter.js");
   |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: couldn't read cli/src/../../lib/binding_web/tree-sitter.wasm: No such file or directory (os error 2)
  --> cli/src/web_ui.rs:16:33
   |
16 | const LIB_WASM: &'static [u8] = include_bytes!("../../lib/binding_web/tree-sitter.wasm");
   |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

error: Could not compile `tree-sitter-cli`.
```

**Comment by maxbrunsfeld** [comment]
Yeah, you currently have to run `script/build-wasm` from the root directory first, because we embed the WASM library inside of the CLI in order to implement the `web-ui` command.

There's some docs here: http://tree-sitter.github.io/tree-sitter/contributing#building

I'm probably going to make this step optional too. I'd like to *not* require the WASM library be built first, and to fetch it from a remote URL if it is not built. But I'm not sure how I'm going to set this up; maybe a cargo feature flag.

**Comment by calixteman** [comment]
Ok, maybe you could add an helpful error message with the url when the file is not found.

**Comment by osa1** [comment]
> you currently have to run script/build-wasm from the root directory first

Would be good to mention this in `cli/README.md`, it's a bit hard to discover currently.

**Comment by maxbrunsfeld** [comment]
Fixed in #723.
